app/assets/stylesheets/helpers/_old-ie.scss

```diff
@@ -0,0 +1,42 @@
+// static specific styles
+#wrapper,
+.survey-wrapper,
+.govuk-emergency-banner div {
+  width: 960px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.js-enabled .govuk-sticky-element--hidden {
+  // XXX: Would be nice to `@include .visuallyhidden`.
```

Comment:
I think this note needs a `TODO` or a `FIXME` - I don't understand what `XXX` means.

Reply:
Me neither! I moved this line of code from somewhere else without modifying it. Probably should be more descriptive. Will change.

Reply:
Turns out thanks to the changes in the gem we can now do what this comment aspired to - include a visually hidden mixin from `govuk-frontend`. Hooray! 😄